### PR TITLE
fix:  Solved issue with method recordManualImpression for GAMBannerAd

### PR DESCRIPTION
**Pull request by amolrajoba**
### Description
fix: for method ```recordManualImpression```  in ReactNativeGoogleMobileAdsBannerAdViewManager.java  
The Android Java code in ReactNativeGoogleMobileAdsBannerAdViewManager.java is expecting an Integer after ```Integer.parseInt``` but currently string 'recordManualImpression' is passed. 

### Checklist

- I read the [Contributor Guide](https://github.com/invertase/react-native-google-mobile-ads/blob/main/CONTRIBUTING.md)
  and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [ ] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `__tests__e2e__`
  - [ ] `jest` tests added or updated in `__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

---

Think `react-native-google-mobile-ads` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

**Comment by amolrajoba** [comment]
> 
for iOS there is no comparison with the passed value in  "RNGoogleMobileAdsBannerViewManager.m" file. In Android there is 
comparison with the passed value after Integer.parseInt in "ReactNativeGoogleMobileAdsBannerAdViewManager.java".
```
    int commandIdInt = Integer.parseInt(commandId);

    if (commandIdInt == COMMAND_ID_RECORD_MANUAL_IMPRESSION) {
      BaseAdView adView = getAdView(reactViewGroup);
      if (adView instanceof AdManagerAdView) {
        ((AdManagerAdView) adView).recordManualImpression();
      }
    }
```
this kind of check not there in iOS code before native method call recordManualImpression() .

**Review by dylancom — CHANGES_REQUESTED** [review]
There are methods available to get the command id.

iOS: UIManager.getViewManagerConfig('RNGoogleMobileAdsBannerView').Commands.recordManualImpression
Android: UIManager.RNGoogleMobileAdsBannerView.Commands.recordManualImpression.toString()

See:
https://reactnative.dev/docs/native-components-ios#handling-multiple-native-views
https://reactnative.dev/docs/native-components-android#6-implement-the-javascript-module

File: src/ads/GAMBannerAd.tsx [hunk 1]
```diff
@@ -22,11 +22,12 @@ import { BaseAd, GoogleMobileAdsBannerView } from './BaseAd';
 
 export class GAMBannerAd extends React.Component<GAMBannerAdProps> {
   private ref = createRef<GoogleMobileAdsBannerView>();
+  private commonIDRecordManualImpression = "1";
 
   recordManualImpression() {
     UIManager.dispatchViewManagerCommand(
       findNodeHandle(this.ref.current),
-      'recordManualImpression',
+      this.commonIDRecordManualImpression,
```
Comment (dylancom) [inline src/ads/GAMBannerAd.tsx:30]:
```suggestion
      Platform.OS === 'ios' ? 'recordManualImpression' : UIManager.RNGoogleMobileAdsBannerView.Commands.recordManualImpression.toString(),
```

**Review by dylancom — CHANGES_REQUESTED** [review]

**Review by mikehardy — APPROVED** [review]
Appears to match the docs based on my read of the docs vs this code, nice fix!

**Comment by ds8k** [comment]
This change is causing the following redbox on simulator:
Screenshot 2023-05-02 at 4 45 45 PM

**Comment by dylancom** [comment]
@ds8k thanks for the report! Should be fixed in 10.3.2

**Pull request merged**